apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.ts

```diff
@@ -76,6 +79,29 @@ export class EventTypesController_2024_06_14 {
     };
   }
 
+  @Post("/:eventTypeId/create-phone-call")
+  @Permissions([EVENT_TYPE_WRITE])
+  @UseGuards(ApiAuthGuard)
+  async createPhoneCall(
+    @Param("eventTypeId") eventTypeId: number,
+    @Body() body: CreatePhoneCallInput,
+    @GetUser() user: UserWithProfile
+  ): Promise<CreatePhoneCallOutput> {
+    const data = await handleCreatePhoneCall({
+      user: {
+        id: user.id,
+        timeZone: user.timeZone,
+        profile: { organization: { id: user.profiles?.[0]?.organization?.id } },
+      },
+      input: { ...body, eventTypeId },
+    });
+
+    return {
+      status: SUCCESS_STATUS,
+      data,
+    };
+  }
+
```

Comment:
can we add en e2e test in apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.e2e-spec.ts